### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JakubPetriska**
Whole app

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <!DOCTYPE html>
-<html>
+<html class="md-scrollbar">
```
Comment (novellizator) [inline index.html:2]: quibble: wasn't that supposed to be rather appended to body element ?

File: src/App.vue [hunk 2]
```diff
@@ -1,28 +1,131 @@
...
+      }
+    }
+  },
+  computed: {
+    availableTimezonesList() {
+      return Array.from(this.availableTimezones.values())
+        .sort((left, right) => left.offset - right.offset)
+    },
+    notAddedTimezones() {
+      return this.availableTimezonesList.filter(e => this.shownTimezones.indexOf(e.value) === -1)
```
Comment (novellizator) [inline src/App.vue]: I believe you may use there this.shownTimezones.includes() which returns a boolean

File: src/main.js [hunk 3]
```diff
@@ -1,10 +1,25 @@
 import Vue from 'vue'
 import App from './App'
```
Comment (novellizator) [inline src/main.js]: convention: first import libs from 3-rd party npm packages, only then import your files. The Only Correct Import order (tm) is: 1. built-in libraries 2. 3-rd party libs 3. internal project libs

File: test/unit/specs/App.spec.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
+import App from '@/App'
```
Comment (novellizator) [inline test/unit/specs/App.spec.js]: import order conventions

File: test/unit/specs/TimezoneCard.spec.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+import TimezoneCard from '@/components/TimezoneCard'
+import TestUtils from '../TestUtils.js'
```
Comment (novellizator) [inline test/unit/specs/TimezoneCard.spec.js]: import order

**Review by novellizator — COMMENTED** [review]
I'm not very familiar with vue.js, but I found a few minor quibbles. Well done

File: index.html (hunk 1 above)
Comment (JakubPetriska) [reply index.html:2]: It was done according to [this](https://github.com/marcosmoura/vue-material/issues/46) and it works. I agree other approaches mentioned in the issue look better but I had trouble making them work so I stick to this one. I could not find mention about this in official documentation.

File: test/unit/specs/App.spec.js (hunk 4 above)
Comment (JakubPetriska) [reply test/unit/specs/App.spec.js]: Fixed in https://github.com/JakubPetriska/timezones/pull/1/commits/d47add8953d8eae4902b1b5457fd10f22757ce29

**Pull request merged**